src/main/java/com/mateacademy/homework10/REUtil.java

```diff
@@ -0,0 +1,32 @@
+package com.mateacademy.homework10;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class REUtil {
+
+    public static boolean validator(String validatorType, String string) {
```

Comment:
better make separate methods for each case + declare regular expressions as constants